sum.c

```diff
@@ -0,0 +1,39 @@
+/* Amanda Cuevas - This program allows the user to input a length and then it will create an array featuring the cumulative sums of them based on numbers inputted by the user */
+#include <stdio.h>
+#include <math.h>
+int main()
+	   {
+		   int length;
+		   int array[length];
```

Comment:
Also, when you hand in code that doesn't compile or run, you should always comment throughout the code.